RELEASING.md

```diff
@@ -31,11 +31,11 @@
    ```
    This will generate the `.deb`, `.rpm` and brew formula files, which will be located in `packaging/builds/X.Y.Z/`.
 
-10. Clone the `Shopify/homebrew-shopify` repository (if not already cloned), and then
+10. Clone the `Shopify/homebrew-shopify` repository (if not already cloned) with `dev clone homebrew-shopify`, and then
```

Comment:
I don't think `Shopify/homebrew-shopify` is a private repo (I think it has to be public for `brew tap` to work), which means that anyone could contribute to it(?), which also means anyone outside of Shopify will not have `dev` available to them.  Hence I didn't specify how to clone.  Granted it's _highly_ unlikely that anyone external will be releasing a new version anyway; most likely we'll always control that aspect of it.